std/stdio.d

```diff
@@ -3283,6 +3289,9 @@ $(LINK2 std_conv.html, to!(string)(arg))) and write the resulting
 string to $(D args[0]). A call without any arguments will fail to
 compile.
 
+Params:
+    args = the items to write to `stdout`
+
```

Comment:
This is unnecessary. It gives me no useful information. I'd rather just leave it out.